core/trino-parser/src/main/java/io/trino/sql/parser/AstBuilder.java

```diff
@@ -3057,6 +3058,8 @@ public Node visitOver(SqlBaseParser.OverContext context)
     @Override
     public Node visitColumnDefinition(SqlBaseParser.ColumnDefinitionContext context)
     {
+        Optional<Literal> defaultValue = visitIfPresent(context.literal(), Literal.class);
```

Comment:
Are you sure it can't match to the string in the `COMMENT` clause, if present? 
Please add a test or manually disambiguate by assigning a label in the grammar: 
`... DEFAULT default_value=literal ...`, 
and then
`... visitIfPresent(context.default_value, Literal.class);`